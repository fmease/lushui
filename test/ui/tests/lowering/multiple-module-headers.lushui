;;; TEST fail check

module

module

module sub of

module

