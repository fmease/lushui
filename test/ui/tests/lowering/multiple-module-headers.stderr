error[E041]: the module header has to be the first declaration of the module
 --> ${TEST_FOLDER}/lowering/multiple-module-headers.lushui:5:1
  |
5 | module
  | ^^^^^^ 
  |
 note: however, the current module already has a module header

error[E041]: the module header has to be the first declaration of the module
 --> ${TEST_FOLDER}/lowering/multiple-module-headers.lushui:9:1
  |
9 | module
  | ^^^^^^ 
  |
 note: however, the current module already has a module header

error: aborting due to 2 previous errors

